### PR TITLE
Add apihistory options

### PR DESCRIPTION
**Pull request by alexose**
This fixes routing for local development.

### CODE REVIEW

File: script/build.js [hunk 1]
```diff
@@ -168,7 +168,9 @@ if (options.watch) {
     contentBase: `build/${options.buildtype}`,
     historyApiFallback: {
       rewrites: [
-        { from: '^\/rx(.*)', to: '/rx/' }
+        { from: '^\/rx(.*)', to: '/rx/' },
+        { from: '^\/healthcare\/apply\/application(.*)', to: '/healthcare/apply/application/' },
```
Comment (jbalboni) [inline script/build.js]: Education benefits is using education/apply-for-education-benefits/apply as the entry page right now.

`{ from: '^\/education\/apply-for-education-benefits\/apply(.*)', to: '/education/apply-for-education-benefits/apply/' },`, maybe?

File: script/build.js [hunk 2]
```diff
@@ -195,7 +197,7 @@ if (options.watch) {
       '/api/*': {
         target: `https://${api.host}${api.path}`,
         auth: api.auth,
-        rewrite: function rewrite(req) {
+        rewrite: function (req) {
```
Comment (jbalboni) [inline script/build.js]: This might be a lint error?

Comment (alexose) [inline script/build.js]: We aren't linting build.js yet... separate PR maybe?

**Comment by jbalboni** [comment]
This page might not be correct: http://localhost:3000/employment/employers/

Otherwise, LGTM.

**Pull request merged**